src-tauri/src/main.rs

```diff
@@ -72,12 +81,59 @@ async fn control_grabbing(
     Ok(())
 }
 
+/// Engages the logger.
+fn engage_logger() -> Result<log4rs::Handle, SetLoggerError> {
+    let level: log::LevelFilter = option_env!("RUST_LOG")
+        .and_then(|s| log::LevelFilter::from_str(s).ok())
+        .unwrap_or(log::LevelFilter::Warn);
```

Comment:
I think you'll want to have the log level for the console controlled by this env, but for the file, you should have it at a pre-defined level which includes more. Especially cus if you just default to Warn, you're not really going to get that many of the logs you're looking for in the log file for debugging.

Maybe for the file you could default to Info, then maybe go up to Debug or Trace if the env defines it to those?

Reply:
May also want to consider at some point adjusting this level filter by verbose cli args, `e.g. -v or -vv` to enable higher levels of logging

Reply:
Looks to still be broken for me

Reply:
Try it now :D should work